NOTICE.txt

```diff
@@ -1605,6 +1716,19 @@ SOFTWARE.
 
 ---
 
+## react-native-file-viewer
+
+This product contains 'react-native-file-viewer' by Vincenzo Scamporlino.
+
+Native file viewer for React Native. Preview any type of file supported by the mobile device.
+
+* HOMEPAGE:
+  * https://github.com/vinzscam/react-native-file-viewer
+
+* LICENSE:
```

Comment:
No license in their repo

Reply:
I'd say lets add an issue in their repo and ask for the license, if it ends up being not compatible then we can remove it and look for an alternative. for now we can keep.

Reply:
@grundleborg Please help confirm if this is the best next step here.

Reply:
I can't really comment. Perhaps ask @wiersgallak for advice?

Reply:
License request: https://github.com/vinzscam/react-native-file-viewer/issues/79.

Reply:
License was added.